src/mui/layout/Breadcrumbs.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import inflection from 'inflection';
+import translate from '../../i18n/translate';
+import Breadcrumb from '../layout/Breadcrumb';
+
+class Breadcrumbs extends Component {
```

Comment:
I guess it should be Breadcrumb (it's usually used at the singular) for that one and find something else for the other one like subBreadcrum

Reply:
Not really, because in Breadcrumbs you have one or two breadcrumbs (according to where you are), and currently Breadcrumb is actually only one breadcrumb. Hence the BreadcrumbsGroup proposal (as there are several of them)